packages/@react-stately/selection/src/types.ts

```diff
@@ -91,7 +96,7 @@ export interface MultipleSelectionManager extends FocusState {
    * Toggles, replaces, or extends selection to the given key depending
    * on the pointer event and collection's selection mode.
    */
-  select(key: Key, e?: PressEvent | LongPressEvent | PointerEvent): void,
+  select(key: Key, eventType?: SelectEventType): void,
```

Comment:
Are we/should we be concerned about the change in api here? People who used SelectionManager or any of our stately hooks that returned a SelectionManager maybe affected by this change here if they called `manager.select(key, e)` anywhere right? Maybe a non-issue if we expect them not to use the selection manager directly and to rely on hooks like `useSelectableItem`